rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ macro(build_ogre)
   list(APPEND extra_cmake_args "-DOGRE_BUILD_RENDERSYSTEM_D3D11:BOOL=OFF")
   list(APPEND extra_cmake_args "-DOGRE_BUILD_RENDERSYSTEM_D3D9:BOOL=OFF")
   if(APPLE)
-    list(APPEND extra_cmake_args "-DCMAKE_OSX_ARCHITECTURES='x86_64'")
+    list(APPEND extra_cmake_args "-DCMAKE_OSX_ARCHITECTURES='x86_64'"
+      "-DCMAKE_CXX_FLAGS=-std=c++14 -stdlib=libc++")
```

Comment:
Hmm, this is strange too, the `-stdlib` should not be needed and the `-std=c++14` is set just a few lines above already.

Reply:
Yet... it does seem needed, I think we should report this upstream. They must be setting the `-stdlib` to something wrong unless we override it or something. There's no other reason that this file `OgreProperty.h` would be able to do `#include <functional>` but not use `std::function` later.

Reply:
Might be something my patch was fixing before:

https://github.com/ros2/rviz/pull/127/files#diff-d20b3504260c943b0679a45f137ff635L28